server/controllers/bookController.js

```diff
@@ -90,6 +90,50 @@ class BookController {
       Utils.errorStatus(res, 500, error.message);
     }
   }
+
+  static async lendBook(req, res) {
+    const { isbn, title, patronId, cost} = req.body;
+    
+    try {
+      const booksNotReturned = await models.Borrowed_books.findAndCountAll({ where: { patronId, returned : false } });
+
+      if (booksNotReturned.count >= 3) {
+        return Utils.errorStatus(res, 405, 'You cannot have more than 3 books in your possession');        
```

Comment:
This error should be `'You cannot have 3 or more books in your possession' because you are checking if it 3 or higher

Reply:
The user can have a maximum of 3 books in their possession so the message should suffice.

Reply:
Yes, But I was referring to the error message itself. It says you cannot have more than 3 books, but you can not have 3 books as well.

Reply:
@PelumiAlesh I think it's cool. You can have 3. But not more

Reply:
@chuxmykel He is using **>=** in ` if (booksNotReturned.count >= 3)` which means you can not have 3.

Reply:
How @PelumiAlesh? Isn't that supposed to be greater than or equal 3? Which means 3 or above now?

Reply:
It means the books he has not returned are 3 or more. So why do you think he can't have 3?

Reply:
@PelumiAlesh by the time that if condition evaluates to true, the user would have had 3 books already. That's when the response is returned.

Reply:
@chuxmykel explained to me. Yeah its good, sorry for the confusion :)